src/commonMain/kotlin/exchange.dydx.abacus/calculator/V2/TradeInput/TradeInputMarketOrderCalculator.kt

```diff
@@ -118,11 +119,28 @@ internal class TradeInputMarketOrderCalculator() {
                 maxMarketLeverage
             }
 
+            val positions = subaccount.openPositions
+            val marketId = market?.perpetualMarket?.id
```

Comment:
it looks like you are getting `marketId` from `trade.marketId` in the non-typed `TradeInputCalculator`. I'm assuming its interchangeable

Reply:
ah yeah - they should be interchangeable, but lemme make them consistent for easier to read reasons